### PR TITLE
Added Abstract Class example

### PR DESCRIPTION
**Pull request by osama10**
I have added the example of creating An abstract class in Python using ABC package

### CODE REVIEW

File: step6/index.py [hunk 1]
```diff
@@ -71,6 +71,63 @@ def info(self):
 dog.set_owner('Donald Trump')
 print(dog.get_owner())
 
+# Abstract Class
+from abc import ABCMeta, abstractmethod
```
Comment (abdulmoizeng) [inline step6/index.py]: @osama10 could you please write some description on ` Abstract Base Classes (ABCs)` for beginners

File: step6/index.py [hunk 2]
```diff
@@ -103,5 +160,4 @@ def full_name(self):
 del emp1.full_name
```
Comment (abdulmoizeng) [inline step6/index.py]: @osama10 could you please write some description on `del` key word usage

File: step6/index.py (hunk 1 above)
Comment (osama10) [reply step6/index.py]: will do .

File: step6/index.py (hunk 2 above)
Comment (osama10) [reply step6/index.py]: will do

**Comment by osama10** [comment]
I have added the required descriptions of ABCs and del keyword . Please review it . Thanks .

File: step6/index.py [hunk 3]
```diff
@@ -71,6 +71,70 @@ def info(self):
...
+    
+    name = property(get_name, set_name)
+
+
+class Man(AbstractHuman):
+    
+    def __init__(self, name):
+        self._name = name
+    
+    def name(self):
```
Comment (abdulmoizeng) [inline step6/index.py:114]: @osama10 why did we need this?

**Comment by abdulmoizeng** [comment]
@all-contributors please add @osama10  for code

**Pull request merged**